tests/postgres_tests/test_aggregates.py

```diff
@@ -110,6 +118,23 @@ def test_string_agg_empty_result(self):
         values = AggregateTestModel.objects.aggregate(stringagg=StringAgg('char_field', delimiter=';'))
         self.assertEqual(values, {'stringagg': ''})
 
+    @skipUnlessDBFeature('has_jsonb_datatype')
+    def test_json_agg(self):
+        """
+        Aggregation for a single value JSON field.
+        """
+        values = AggregateTestModel.objects.aggregate(jsonagg=JsonAgg('char_field'))
+        self.assertEquals(values, {'jsonagg': ['Foo1', 'Foo2', 'Foo3', 'Foo4']})
+
+    @skipUnlessDBFeature('has_jsonb_datatype')
+    def test_empty_json_agg(self):
+        """
+        Test aggregation over an empty queryset.
+        """
+        AggregateTestModel.objects.all().delete()
+        values = AggregateTestModel.objects.aggregate(jsonagg=JsonAgg('integer_field'))
```

Comment:
You could avoid the delete query with: `AggregateTestModel.objects.none().aggregate(...)`. (I guess the other tests could use the same pattern, not for this PR though.)